package.json

```diff
@@ -18,9 +18,9 @@
     "lint": "npm run lint:format && npm run lint:style",
     "lint:fix": "npm run lint:format:fix && npm run lint:style:fix",
     "build": "tsc -p tsconfig.build.json",
-    "test": "jest test/unit",
+    "test": "jest test/unit --forceExit",
     "test:watch": "jest test/unit --watch",
-    "test:coverage": "jest test/unit --coverage",
+    "test:coverage": "jest test/unit --coverage --forceExit",
```

Comment:
@gustavobeavis the test pipeline was not finishing, probably because the grpc server that was keeping the process up, so I forced exit it